model/__init__.py

```diff
@@ -191,6 +191,9 @@ def to_module_info(mi):
             kwargs['name'] = cfg_name
 
         element_instance = element_type(**kwargs)
+        validate_function = getattr(element_instance, "validate", None)
```

Comment:
just call the method. if an instance should not have it, this is an error (which we shall find with the tests you are adding with this change :-))

Reply:
done